pkgs/development/tools/build-managers/gradle/default.nix

```
@@ -51,7 +51,17 @@ rec {
     };
   };
 
-  gradle_latest = gradle_5_6;
+  gradle_latest = gradle_6_5;
+
+  gradle_6_5 = gradleGen rec {
+    name = "gradle-6.5.1";
+    nativeVersion = "0.22";
+
+    src = fetchurl {
+      url = "http://services.gradle.org/distributions/${name}-bin.zip";
```

```suggestion
      url = "https://services.gradle.org/distributions/${name}-bin.zip";
```

I get a 403 for that URL as well, but if I change this to use https it works:

```
❯ nix-prefetch-url http://services.gradle.org/distributions/gradle-6.5.1-bin.zip 
[0.0 MiB DL]
error: unable to download 'http://services.gradle.org/distributions/gradle-6.5.1-bin.zip': HTTP error 403
❯ nix-prefetch-url https://services.gradle.org/distributions/gradle-6.5.1-bin.zip
[97.6 MiB DL]
path is '/nix/store/fk10bkypfhjnsgsc2b2lyxxpnqpghs31-gradle-6.5.1-bin.zip'
0jmmipjh4fbsn92zpifa5cqg5ws2a4ha0s4jzqhrg4zs542x79sh
```

```